src/GUI/Overlay/ImGuiWrapper.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+class ImGuiWrapper {
+ public:
+    ImGuiWrapper();
+    ~ImGuiWrapper();
+};
```

Comment:
I'm not happy with this class name. It takes care of initializing an imgui context. In the next branch I'm also intiializing the style/theme and other minor things. If you have a better name please tell me.

Reply:
Shouldn't we just do the same thing we did with Nuklear?

Hide all (most?) of ImGui-related calls inside an `ImGuiRenderer`, create it from inside the `OpenGLRenderer`, and don't even include any of ImGui headers from `OpenGLRenderer.cpp`?

Reply:
I thought to keep the calls directly in the OpenGLRenderer.cpp considered that the amount of calls are fewer than `nuklear`. If things starts to grow I can move them to another class later.